<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:#131239;</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>275</x>
     <y>190</y>
     <width>250</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 25pt &quot;Georgia&quot;;
color:white;</string>
   </property>
   <property name="text">
    <string>Your Score :</string>
   </property>
  </widget>
  <widget class="QLabel" name="f_score">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>270</y>
     <width>200</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20pt &quot;Georgia&quot;;
color:white;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="f_playagain">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>420</y>
     <width>200</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;Arial Rounded MT Bold&quot;;
padding:15px;
background-color:green;
color:white;

</string>
   </property>
   <property name="text">
    <string>Play Again</string>
   </property>
  </widget>
  <widget class="QPushButton" name="f_next">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>540</y>
     <width>200</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;Arial Rounded MT Bold&quot;;
padding:15px;
background-color:green;
color:white;

</string>
   </property>
   <property name="text">
    <string>Next</string>
   </property>
  </widget>
  <widget class="QPushButton" name="f_quit">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>650</y>
     <width>200</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;Arial Rounded MT Bold&quot;;
padding:15px;
background-color:green;
color:white;

</string>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
  </widget>
  <widget class="QLabel" name="result_label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>80</y>
     <width>350</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 25pt &quot;Georgia&quot;;
color:white;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="result_name">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>80</y>
     <width>350</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 25pt &quot;Georgia&quot;;
color:white;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
